timezone.gemspec

```diff
@@ -14,19 +13,23 @@ Gem::Specification.new do |s|
   s.summary     = "timezone-#{Timezone::VERSION}"
   s.license     = 'MIT'
   s.description = 'Accurate current and historical timezones for Ruby with ' \
-    'support for Geonames and Google latitude - longitude lookups.'
+                  'support for Geonames and Google latitude - longitude ' \
+                  'lookups.'
 
   s.files       = `git ls-files`.split("\n")
-  s.test_files  = `git ls-files -- {test,spec,features}/*`.split("\n")
```

Comment:
How come this is removed?

Reply:
https://docs.rubocop.org/rubocop/cops_gemspec.html#gemspecdeprecatedattributeassignment - because it an unused argument at this point.